src/Api/MergeRequests.php

```diff
@@ -61,6 +61,8 @@ class MergeRequests extends AbstractApi
      *     @var string             $labels         return merge requests matching a comma separated list of labels
      *     @var \DateTimeInterface $created_after  return merge requests created after the given time (inclusive)
      *     @var \DateTimeInterface $created_before return merge requests created before the given time (inclusive)
+     *     @var int                $reviewer_id    returns merge requests which have the user as a reviewer with the given user id
+     *     @var string             $wip            return only draft merge requests (yes) or only non-draft merge requests (no)
```

Comment:
I think this should be a boolean, and be transformed in the method. Can you change this please?

Reply:
Heyho! I've changed it. Hope, this is okay, if not let me know :)

Thanks!